packages/@o3r/training-sdk/package.json

```diff
@@ -0,0 +1,138 @@
+{
+  "name": "@o3r/training-sdk",
+  "private": true,
+  "version": "0.0.0-placeholder",
+  "description": "training-sdk - SDK",
+  "source": "src/index.ts",
+  "main": "cjs/index.js",
+  "module": "esm2020/index.js",
+  "node": "cjs/index.js",
+  "es2020": "esm2020/index.js",
+  "esm2020": "esm2020/index.js",
+  "es2015": "esm2015/index.js",
+  "esm2015": "esm2015/index.js",
+  "types": "index.d.ts",
+  "sideEffects": false,
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    "./structure/src.json": {
+      "default": "./structure/src.json"
+    },
+    "./structure/spec.json": {
+      "default": "./structure/src.json"
+    },
+    ".": {
+      "main": "./cjs/index.js",
+      "module": "./esm2020/index.js",
+      "es2020": "./esm2020/index.js",
+      "esm2020": "./esm2020/index.js",
+      "node": "./cjs/index.js",
+      "default": "./esm2020/index.js",
+      "typings": "./index.d.ts",
+      "import": "./esm2020/index.js",
+      "require": "./esm2020/index.js"
+    }
+  },
+  "scripts": {
+    "schematics": "schematics",
+    "clean": "rimraf test/ test-dev/ dist/ dist-dev/ dist-test/ build/",
+    "lint:ci": "eslint \"**/*[jt]s\" --quiet --format junit --output-file ./dist-lint/result.xml",
+    "lint": "eslint \"**/*[jt]s\" --cache",
+    "start": "tsc-watch -b tsconfigs/esm2020 --noClear --onFirstSuccess \\\"yarn run files:pack --watch\\\"",
+    "extract": "yarn run extract-src && yarn run extract-openapi",
+    "extract-src": "o3r-extract-folder-structure --files \"src\" -o dist/structure/src.json",
+    "extract-openapi": "o3r-extract-folder-structure --files \"./open-api.yaml\" -o dist/structure/spec.json",
+    "build": "yarn run build:cjs && yarn run build:esm2015 && yarn run build:esm2020 && yarn run files:pack && yarn run extract",
+    "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q --strip-leading-paths",
+    "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q --strip-leading-paths",
+    "build:esm2020": "tsc -b tsconfigs/esm2020",
+    "set:version": "npm version",
+    "resolve": "node -e 'process.stdout.write(require.resolve(process.argv[1]));'",
+    "generate": "schematics ../../@ama-sdk/schematics:typescript-core --no-dry-run",
+    "spec:regen": "yarn run generate --generator-key ama-sdk-training-sdk && amasdk-clear-index",
+    "spec:upgrade": "yarn run spec:regen",
+    "files:pack": "amasdk-files-pack",
+    "test": "jest --passWithNoTests",
+    "publish:package": "npm publish ./dist",
+    "generate:mock": "schematics @ama-sdk/schematics:mock",
+    "doc:generate": "typedoc",
+    "tools:changelog": "commit-and-tag-version"
+  },
+  "lint-staged": {
```

Comment:
not needed, I suppose you keep it for example purpose but I wonder if it will not lead to issue.
(the generator should not generate it if the SDK is part of a monorepo)

Reply:
this was automatically generated, but I may have done the generation wrong... I will remove this